app/controllers/course_summaries_controller.rb

```diff
@@ -0,0 +1,11 @@
+class CourseSummariesController < ApplicationController
+  include CourseSummariesHelper
+
+  def index
+    @assignments = Assignment.all();
```

Comment:
Do not use parentheses for method calls with no arguments.<br>Do not use semicolons to terminate expressions.